en/docs/index.md

```diff
@@ -28,27 +28,20 @@ template: templates/single-column.html
             <div class="linkSet2" onclick="location.href='get-started/quick-start-guide';">
                 <a href="get-started/quick-start-guide"><h3>Quick Start Guide</h3></a>
                 <p>
-                    If you're new to the WSO2 APK platform, the quick start guide is a great place to get started. This guide will walk you through the process of setting up and running your first API on the platform.
+                    The Quick Start Guide is the perfect introduction to get started with WSO2 APK platform. This guide provides step-by-step instructions on how to set up APK in your working environment and run your first API on the platform. Whether you're new to the platform or looking to explore its features, the Quick Start Guide is an excellent resource to help you get up and running quickly.
                 </p>
             </div>
-            <!--<div class="linkSet2 middle" onclick="location.href='../restapi/overview';">
-                <img src="assets/img/home/landing-page/write-your-first-integration-service.svg" title="Lorem ipsum" width="75" alt="Lorem ipsum"/>
-                <a href="/get-started/quick-start-guide"><h3>Basic Flow</h3></a>
-                <p>
-                    The basic flow guide for the WSO2 APK platform provides step-by-step tutorials for developing, managing and publishing APIs.
-                </p>
-            </div>-->
+            </div>
             <div class="linkSet2 middle" onclick="location.href='about-apk/what-is-apk';">
                 <a href="about-apk/what-is-apk"><h3>What is APK?</h3></a>
                 <p>
-                    <!--The basic flow guide for the WSO2 APK platform provides step-by-step tutorials for developing, managing and publishing APIs.-->
-                    Get to know WSO2 APK one step deeper.
+                    WSO2 API Manager (APK) provides a complete solution for API management, covering the entire API lifecycle from design and development to publishing and consumption. With its support for API gateway, developer portal, and marketplace capabilities, the platform offers a robust set of tools for managing APIs. Additionally, WSO2 APK includes key architectural features specifically designed for Kubernetes environments, making it easy to deploy and manage APIs in a cloud-native way.
```

Comment:
API Manager referal intentional?

Reply:
This need to correct. Thanks for pointing